src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -13,53 +22,46 @@ import {
-import { VisualizationPlugin } from '../../core/components/visualizations/VisualizationPlugin';
-import { LayoutOptions } from '../../core/components/layouts/types';
-import { OverlayOptions } from '../../core/components/visualizations/options/types';
+import FilterChipList from '../../core/components/FilterChipList';
 import { FloatingLayout } from '../../core/components/layouts/FloatingLayout';
+import { LayoutOptions } from '../../core/components/layouts/types';
+import { VariableLinkConfig } from '../../core/components/VariableLink';
+import {
+  useFeaturedFields,
+  useFieldTree,
+  useFlattenedFields,
+} from '../../core/components/variableTrees/hooks';
+import { barplotVisualization } from '../../core/components/visualizations/implementations/BarplotVisualization';
+import { boxplotVisualization } from '../../core/components/visualizations/implementations/BoxplotVisualization';
+import { histogramVisualization } from '../../core/components/visualizations/implementations/HistogramVisualization';
+import { lineplotVisualization } from '../../core/components/visualizations/implementations/LineplotVisualization';
 import {
   contTableVisualization,
   twoByTwoVisualization,
 } from '../../core/components/visualizations/implementations/MosaicVisualization';
 import { scatterplotVisualization } from '../../core/components/visualizations/implementations/ScatterplotVisualization';
-import { lineplotVisualization } from '../../core/components/visualizations/implementations/LineplotVisualization';
-import { barplotVisualization } from '../../core/components/visualizations/implementations/BarplotVisualization';
-import { boxplotVisualization } from '../../core/components/visualizations/implementations/BoxplotVisualization';
+import { OverlayOptions } from '../../core/components/visualizations/options/types';
+import { VisualizationPlugin } from '../../core/components/visualizations/VisualizationPlugin';
 import ShowHideVariableContextProvider from '../../core/utils/show-hide-variable-context';
-import { MapLegend } from './MapLegend';
-import { AppState, useAppState } from './appState';
-import { FloatingDiv } from './FloatingDiv';
 import Subsetting from '../../workspace/Subsetting';
 import { findFirstVariable } from '../../workspace/Utils';
-import {
-  useFeaturedFields,
-  useFieldTree,
-  useFlattenedFields,
-} from '../../core/components/variableTrees/hooks';
-import {
-  SemiTransparentHeaderLogoProps,
-  SemiTransparentHeader,
-} from './SemiTransparentHeader';
-import FilterChipList from '../../core/components/FilterChipList';
-import { VariableLinkConfig } from '../../core/components/VariableLink';
+import { AppState, useAppState } from './appState';
+import { FloatingDiv } from './FloatingDiv';
+import { MapLegend } from './MapLegend';
+import { SemiTransparentHeader, SiteProps } from './MapNavigation';
+import { MapSideNavigation } from './MapSideNavigation';
```

Comment:
My apologies... I saw an `Optimize imports` command in VS Code and forgot to undo it after see what it'd do.